examples/resources/ekscluster/cluster.tf

```diff
@@ -50,19 +50,16 @@ resource "tanzu-mission-control_ekscluster" "tf_eks_cluster" {
         ]
       }
 
-      addons_config {
+      addons_config { // this whole section is optional
         vpc_cni_config {
           eni_config {
-            id = "subnet-0a680171b6330619f" // Required, should belong to the same VPC as the cluster
-            security_groups = [
+            id = "subnet-0a680171b6330619f" // Required, should belong to the same VPC as the cluster, subnets provided in vpc_cni_config are expected to be in different AZs
```

Comment:
It need not be in the same VPC as the Cluster. subnets provided in list of end_config within vpc_cni_config are expected to be in different AZs

Reply:
@DheerajSShetty fixed this